databuilder/extractor/dashboard/looker/looker_view_descriptions_extractor.py

```diff
@@ -96,9 +96,9 @@ def _get_raw_extract_iter(self):
         :return:
         """
         try:
-            file_contents = next(self._github_file_extractor.extract())
-            while file_contents:
-                yield file_contents
-                file_contents = next(self._github_file_extractor.extract())
+            result = self._github_file_extractor.extract()
```

Comment:
this was inconsistent with other extractors which are not generators, although I agree your way was better